### DISCUSSION TITLE
Как реализован whitelist/blacklist через командную строку? Я хочу указать такой же список, как в UI, но через командную строку

### DISCUSSION THREAD

**Discussion by snckmykek** [dovecoteescapee/ByeDPIAndroid#212] [Q&A]
Хочу указать -H (--hosts), но в документации туда нужно указать файл. Когда я запускаю из настроек UI, там можно указать список, а не файл. В итоге всё равно же собирается какая-то командная строка для запуска: кто-нибудь знает, где происходит эта конвертация из UI в консольные команды? Поковырялся в коде, но не нашел ничего.
Или может просто кто-то знает, как мне указать через команду список, на который будет применена программа?

**Comment by anatolysurkov** [comment]
Нужно добавить : перед строкой. Пример --hosts :"googlevideo.com gvt1.com"
